web/libs/editor/src/tags/object/RichText/domManager.ts

```diff
@@ -366,7 +375,11 @@ class DomData {
     let fromIdx = this.displayedTextPos;
     const contentParts = [];
 
-    while (displayedText[fromIdx] === LF) {
+    // it should be just LF but in some OS / browsers it's CRLF (at least)
+    // in the other hand `fromIdx` is only inner counter that does not affect start/end offsets in results
```

Comment:
"in the other hand" is excess here
```suggestion
    // `fromIdx` is only an inner counter that does not affect start/end offsets in results
```